### PR TITLE
Updade/packages/link

### PR DESCRIPTION
**Pull request by LuisConceicaoTD**

### CODE REVIEW

File: packages/atoms/button/src/styles/color-schemes/_elem-link.scss [hunk 1]
```diff
@@ -1,113 +1,77 @@
 $link-light-scheme: (
   "primary": (
     "default": (
-      "color": "text-secondary",
```
Comment (LuisConceicaoTD) [inline packages/atoms/button/src/styles/color-schemes/_elem-link.scss:4]: Should we keep colors we're not using in our component's color scheme?

File: packages/atoms/button/src/styles/color-schemes/_elem-link.scss [hunk 2]
```diff
@@ -1,113 +1,77 @@
...
-      "bg-color": "primary",
-      "brd-color": "primary",
```
Comment (LuisConceicaoTD) [inline packages/atoms/button/src/styles/color-schemes/_elem-link.scss:6]: Should we keep colors we're not using in our component's color scheme?

File: packages/atoms/button/src/styles/color-schemes/_elem-link.scss [hunk 3]
```diff
@@ -1,113 +1,77 @@
...
+      "color": "primary",
     ),
     "hover": (
       "color": "text-secondary",
-      "bg-color": "secondary",
-      "brd-color": "secondary",
```
Comment (LuisConceicaoTD) [inline packages/atoms/button/src/styles/color-schemes/_elem-link.scss:11]: Should we keep colors we're not using in our component's color scheme?

Comment (Mar0cas) [reply packages/atoms/button/src/styles/color-schemes/_elem-link.scss:11]: No, remove what we don't use pls.

**Review by Mar0cas — APPROVED** [review]

**Pull request merged**